main/res/values/strings.xml

```diff
@@ -867,6 +868,7 @@
     <string name="cache_menu_spoilers">Spoiler images</string>
     <string name="cache_menu_around">Caches around</string>
     <string name="around">Around %s</string>
+    <string name="menu_caches">Manage caches &#8230;</string>
```

Comment:
As said before, I would leave out any symbol at this place. The title "Manage..." now speaks for itself and user will click it to see whats behind